app/src/main/java/ch/sdp/vibester/activity/BuzzerSetupActivity.kt

```diff
@@ -86,7 +91,11 @@ class BuzzerSetupActivity : AppCompatActivity(), AdapterView.OnItemSelectedListe
             arrayOf(R.id.namePlayer1, R.id.namePlayer2, R.id.namePlayer3, R.id.namePlayer4)
         var i = 0
         for (playerView in players) {
-            pNameArray[i] = findViewById<EditText>(editTextIdArray[i]).text.toString()
+            val name = findViewById<EditText>(editTextIdArray[i]).text.toString()
+            if (name.isNotEmpty()) { pNameArray[i] = name } else {
```

Comment:
is there a particular reason why the if-else is formatted like this?

Reply:
It was to remain under 25 lines of code for the function for the CodeClimate check 😅

Reply:
ah okkkk, in this case, create a separate function then make a call to it would be a solution too